slides/05-feature-engineering.qmd

```diff
@@ -320,15 +315,14 @@ PCA feature extraction...
 ## Other possible steps `r hexes("recipes", "embed")`
 
 ```{r}
-#| code-line-numbers: "7"
+#| code-line-numbers: "6"
 #| eval: false
-library(embed)
 nhl_rec <- 
   recipe(on_goal ~ ., data = nhl_train) %>% 
   step_dummy(all_nominal_predictors()) %>% 
   step_zv(all_predictors()) %>% 
   step_normalize(all_numeric_predictors()) %>% 
-  step_umap(all_numeric_predictors(), outcome = on_goal)
+  embed::step_umap(all_numeric_predictors(), outcome = on_goal)
```

Comment:
Since we highlight this line, I thought showing `embed::` was a nice way to highlight that this is an extension package